### PR TITLE
feat(handlers/guild): change createGuildChannel() to use guild ID

### PR DESCRIPTION
**Pull request by itohatweb**
closes: #527

### CODE REVIEW

File: test/mod.test.ts [hunk 1]
```diff
@@ -151,7 +151,7 @@ Deno.test({
     const guild = cache.guilds.get(tempData.guildID);
     if (!guild) throw new Error("Guild not found");
 
-    const channel = await createGuildChannel(guild, "test");
+    const channel = await createGuildChannel(guild.id, "test");
```
Comment (ayntee) [inline test/mod.test.ts:154]:
```suggestion
    const channel = await createGuildChannel(tempData.guildID, "test");
```

File: test/mod.test.ts [hunk 2]
```diff
@@ -151,7 +151,7 @@ Deno.test({
     const guild = cache.guilds.get(tempData.guildID);
     if (!guild) throw new Error("Guild not found");
```
Comment (ayntee) [inline test/mod.test.ts:152]:
```suggestion
```

File: test/mod.test.ts [hunk 3]
```diff
@@ -148,10 +148,8 @@ Deno.test({
 Deno.test({
   name: "[channel] create a channel in a guild",
   async fn() {
-    const guild = cache.guilds.get(tempData.guildID);
-    if (!guild) throw new Error("Guild not found");
```
Comment (itohatweb) [inline test/mod.test.ts:152]: why removing that? there we can check indirectly if guild cache is working.

Comment (ayntee) [reply test/mod.test.ts:152]: The test case is not to check whether or not, cache is working.

Comment (itohatweb) [reply test/mod.test.ts:152]: then we should add that in another pr

**Review by Skillz4Killz — APPROVED** [review]

**Review by ayntee — APPROVED** [review]

**Pull request merged**